app/src/main/java/com/innowisegroup/datetimepicker/DateTimePickerDialog.kt

```diff
@@ -201,43 +234,78 @@ class DateTimePickerDialog : DialogFragment() {
     }
 
     companion object {
-        const val UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"
-        const val UPDATE_DATE_TAB_TITLE_REQUEST_KEY = "updateDateTabTitleKey"
+        private const val DIALOG_TAG = "date_time_picker_dialog"
 
-        const val UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
-        const val UPDATE_DATE_TAB_TITLE_KEY = "updateDateTabTitleKey"
+        internal const val UPDATE_TIME_TAB_TITLE_REQUEST_KEY = "updateTimeTabTitleKey"
+        internal const val UPDATE_DATE_TAB_TITLE_REQUEST_KEY = "updateDateTabTitleKey"
 
-        private const val DIALOG_TAG = "date_time_picker_dialog"
+        internal const val UPDATE_TIME_TAB_TITLE_KEY = "updateTimeTabTitleKey"
+        internal const val UPDATE_DATE_TAB_TITLE_KEY = "updateDateTabTitleKey"
 
         private const val INITIAL_LOCAL_DATE_TIME = "initialLocalDateTime"
         private const val MIN_LOCAL_DATE_TIME = "minLocalDateTime"
         private const val MAX_LOCAL_DATE_TIME = "maxLocalDateTime"
         private const val WRAP_SELECTION_WHEEL = "wrapSelectionWheel"
         private const val FRAGMENT_TO_CREATE = "fragmentToCreate"
 
-        private val MIN_DEFAULT_DATE_TIME =
-            LocalDateTime(LocalDate.of(1, 1, 1900), LocalTime.of(0, 0))
-        private val MAX_DEFAULT_DATE_TIME =
-            LocalDateTime(LocalDate.of(1, 1, 2100), LocalTime.of(0, 0))
+        private val MIN_DEFAULT_TIME = LocalTime.of(0, 0)
+        private val MAX_DEFAULT_TIME = LocalTime.of(23, 59)
+        private val MIN_DEFAULT_DATE = LocalDate.of(1, 1, 1900)
+        private val MAX_DEFAULT_DATE = LocalDate.of(1, 1, 2100)
+        private val MIN_DEFAULT_DATE_TIME = LocalDateTime(MIN_DEFAULT_DATE, MIN_DEFAULT_TIME)
+        private val MAX_DEFAULT_DATE_TIME = LocalDateTime(MAX_DEFAULT_DATE, MAX_DEFAULT_TIME)
+
+        @JvmStatic
+        @JvmOverloads
+        fun createTimePickerDialog(
+            initialLocalTime: LocalTime = LocalTime.now(),
+            minLocalTime: LocalTime = MIN_DEFAULT_TIME,
+            maxLocalTime: LocalTime = MAX_DEFAULT_TIME,
+            pickerType: PickerType = PickerType.TIME_ONLY,
```

Comment:
Remove this line pickerType: PickerType = PickerType.TIME_ONLY from constructor. You should not put a value in the constructor, which is explicitly defined for the picker type